lib/galaxy/app.py

```diff
@@ -800,7 +800,7 @@ def __init__(self, **kwargs) -> None:
         self.url_for = url_for
         self.legacy_url_for = legacy_url_for
 
-        self.server_starttime = server_starttime  # used for cachebusting
+        self.start_timestamp = self.server_starttime = server_starttime  # used for cachebusting
```

Comment:
Do we need 2 attributes for this? Not sure if `self.start_timestamp` is needed or we could just use `self.server_starttime` ?

Reply:
Alright! Thanks for the suggestion, I changed it back to `server_startttime` :+1: